SERVER-README.md

```diff
@@ -18,4 +18,9 @@
 ### Workflow
 - ``uwsgi`` is used to run the django application inside docker container
 - ``nginx`` reverse proxy is configured to point the docker django application mapped port
-- static and media files are served by ``nginx`` itself by pointing the url to the static and media directory of the application
+- static and media files are served by ``nginx`` itself by pointing the url to the static and media directory of the application ie. ``/src/static_in_project/static_root`` and ``/src/static_in_project/media_root``
```

Comment:
Static and media files should not be served from the project directly.  They should be somewhere else.  /var/www/bytedeck/media, or something.  This should be indicated in the production settings file.  Collectstatic will place the files where they need to go, right?